catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/CatalogCommands.java

```diff
@@ -119,15 +119,14 @@ protected StringBuilder getUserInputModifiable() throws IOException {
         int in;
         StringBuilder builder = new StringBuilder();
```

Comment:
✏️  This should probably be named `userInput` or something more contextually informative.